_data/rules.yml

```diff
@@ -1300,6 +1300,74 @@ groups:
                 severity: critical
                 for: 2m
 
+      - name: Clickhouse
+        exporters:
+          - name: Embedded Exporter
+            slug: embedded-exporter
+            doc_url: https://clickhouse.com/docs/en/operations/system-tables/metrics
+            rules:
+              - name: ClickHouse Memory Usage Critical
```

Comment:
The memory can be high because of a single query processing.

What about a `for: 3m` parameter?

(same comment for the warning below)

Reply:
Hi @samber,

You are right, it could be even increased to 5 minutes to avoid unnecessary alerts for occasional spikes.